docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,433 @@
+       "| str       |      f64 |       --- |       --- |   |       --- |  thampton |    rbourg |   enstown |\n",
+       "|           |          |       f64 |       f64 |   |       f64 |       --- |       --- |       --- |\n",
+       "|           |          |           |           |   |           |       f64 |       f64 |       f64 |\n",
+       "+==================================================================================================+\n",
+       "| min       |  0.16670 |   0.00000 |   1.00000 | … |   0.00000 |   0.00000 |   0.00000 |   0.00000 |\n",
+       "| max       | 80.00000 |   8.00000 |   3.00000 | … |   1.00000 |   1.00000 |   1.00000 |   1.00000 |\n",
+       "| mean      | 29.88113 |   0.49885 |   2.29488 | … |   0.35600 |   0.69824 |   0.20626 |   0.09396 |\n",
+       "| median    | 29.88113 |   0.00000 |   3.00000 | … |   0.00000 |   1.00000 |   0.00000 |   0.00000 |\n",
+       "| standard  | 12.88320 |   1.04166 |   0.83784 | … |   0.47900 |   0.45920 |   0.40478 |   0.29189 |\n",
+       "| deviation |          |           |           |   |           |           |           |           |\n",
+       "| distinct  | 99.00000 |   7.00000 |   3.00000 | … |   2.00000 |   2.00000 |   2.00000 |   2.00000 |\n",
+       "| value     |          |           |           |   |           |           |           |           |\n",
+       "| count     |          |           |           |   |           |           |           |           |\n",
+       "| idness    |  0.07563 |   0.00535 |   0.00229 | … |   0.00153 |   0.00153 |   0.00153 |   0.00153 |\n",
+       "| missing   |  0.00000 |   0.00000 |   0.00000 | … |   0.00000 |   0.00000 |   0.00000 |   0.00000 |\n",
+       "| value     |          |           |           |   |           |           |           |           |\n",
+       "| ratio     |          |           |           |   |           |           |           |           |\n",
+       "| stability |  0.20092 |   0.68067 |   0.54163 | … |   0.64400 |   0.69824 |   0.79374 |   0.90604 |\n",
+       "+-----------+----------+-----------+-----------+---+-----------+-----------+-----------+-----------+"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
    ],
+   "source": [
+    "transformed_raw_data.summarize_statistics()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Spliting the `raw_data` into train and test sets\n",
+    "- **Training set**: Contains 60% of the data and will be used to train the model.\n",
+    "- **Testing set**: Contains 40% of the data and will be used to test the model's accuracy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_table, test_table = transformed_raw_data.shuffle_rows().split_rows(0.6)"
```

Comment:
Splitting should be done before any kind of learning. This includes the fitting of transformers, like `OneHotEncoder` and `Imputer`. Fit them on the training data and then use the fitted transformers on training and test data. See [this comment](https://datascience.stackexchange.com/a/38441) for example.